records/accessibilities.md

```diff
@@ -0,0 +1,56 @@
+## Overview
+
+We improved the accessibility of SMP to help vision impaired people using it with a screen reader. The main thing used is Accessible Rich Internet Applications (ARIA), which  is a set of roles and attributes that define ways to make web content and web applications (especially those developed with JavaScript) more accessible to people with disabilities. It supplements HTML so that interactions and widgets commonly used in applications can be passed to assistive technologies.
+
+Below listed all the specified behaviours when using a screen reader.
+
+## Burger menu
+| Object | Behaviour | State | Screen Reader Result |
+|-------------|--------|----------|---------|
+| Open burger button | Focus  | Not Clicked | Button collapsed not pressed |
```

Comment:
After chatting with Martin, we've decided that buttons should use *either* `aria-expanded` or `aria-pressed`, but not both at the same time.

This is potentially ambiguous because a lot of our buttons are toggle buttons (which should use `aria-pressed`) that expand or hide various panels when pressed - so which should they use, `aria-pressed` or `aria-expanded`? We've decided to resolve this in the following way:

- Buttons with *explicit* expand behaviour, such as the Expand buttons for expanding the SIP, the story panel and comments in the comments panel, should use `aria-expanded` only.
- All other toggle buttons should use `aria-pressed` only.

This is not the existing implementation as we've used both in a lot of places, but we should specify the desired behaviour for when we come around to do a second pass on this.

In the case of this interaction, the screen reader result should be `Button not pressed`.